packages/sanity/src/core/releases/tool/detail/documentTable/useReleaseHistory.ts

```diff
@@ -76,22 +69,15 @@ export function useReleaseHistory(
 
     await acquireHistorySlot()
     try {
-      const transactions: TransactionLogEventWithEffects[] = []
-      const stream = await getJsonStream(transactionsUrl, token)
-      const reader = stream.getReader()
-
-      const readAll = async (): Promise<void> => {
-        const result = await reader.read()
-        if (result.done) return
-        if ('error' in result.value) {
-          throw new Error(result.value.error.description || result.value.error.type)
-        }
-
-        transactions.push(result.value)
-        await readAll()
-      }
+      const transactions = await getTransactionsLogs(client, versionId, {
+        tag: 'sanity.studio.releases.documents.history',
```

Comment:
I _think_ this name is fine though it's long, let me know if I should rename it something different. Considered 'sanity.studio.releases-documents.history' but felt that was worse and I couldn't find any other instances where we do a '-' in these sort of things